src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozerList = new ArrayList<>();
```

Comment:
Create several objects and add them to the list and also initialize their fields